File: Programmers/Java/2016년.java
package LV1.Java;

class Solution {
    public String solution(int a, int b) {
        String answer = "";
        
        int days = -1;
        if(a > 1) {
            for(int i=1; i<a; i++){
                if(i == 2) days += 29;
                else if(i < 8){
                    if((i%2) == 1) days += 31;
                    else days += 30;
                }
                else {
                    if((i%2) == 1) days += 30;
                    else days += 31;
                }
            }
        }
        days += b;
        
        switch(days%7){
            case 0:
                answer = "FRI";
                break;
            case 1:
                answer = "SAT";
                break;
            case 2:
                answer = "SUN";
                break;
            case 3:
                answer = "MON";
                break;
            case 4:
                answer = "TUE";
                break;
            case 5:
                answer = "WED";
                break;
            case 6:
                answer = "THU";
                break;
        }
        
        return answer;
    }
}

File: Programmers/Python/2022/2016년.py
def solution(a, b):
    weeks = ['THU', 'FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED']
    days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    return weeks[(sum(days[:a-1])+b)%7]